.circleci/config.yml

```diff
@@ -156,6 +156,14 @@ jobs:
           root: *packages_workspace
           paths:
             - "*.tar.gz"
+  docs:
+    <<: *container_config
+    steps:
+      - checkout
+      - run:
+          command: |
+            make swagger-docs
```

Comment:
I'd suggest to move this to the build step and always publish it as CircleCI artifacts.

After that in the `docs` step (I'd also suggest renaming it to `deploy_api_docs` sync the generated docs with the remote docs repo.

Reply:
ok will dig around docs to see how setup this as circle ci artifact

Reply:
@dincho not sure this can work as CircleCi artifacts of i understand this correct, this will store docs in CircleCi, not push them to github.

Reply:
yes, we just want to see the docs for each commit, but publish to gh-pages only master and tags

Reply:
oh ok make sense

Reply:
Never-mind, i don't think it will be usable.

Reply:
@dincho i almoust finished implementation :P

Reply:
cleanup ;]